README.rst

```diff
@@ -1,4 +1,4 @@
-torchvision
+**Torchvision**
```

Comment:
We haven't been consistent with the capitalization of torchvision in the past. We've written it both as torchvision as well as TorchVision. From this perspective, I would maybe leave this as is, but I'm bringing this to discussion with @dongreenberg